.github/workflows/check-format.yml

```diff
@@ -0,0 +1,20 @@
+name: Check for formatting
+
+on:
+  push:
+    branches: [ "**" ]
+
+jobs:
+  check-format:
+    name: Check if code needs formatting
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Setup Maven
+        uses: actions/setup-java@v4
+        with:
+          java-version: 21
+          distribution: zulu
+      - name: Check if code aligns with code style
+        run: mvn spotless:check
```

Comment:
Can we just log out something to the `Workflow Summary`?
Like this
![image](https://github.com/apache/dubbo/assets/9292748/af83c3d6-a44a-42b6-9419-0ce205ae930a)


After that, we can make this job as the first job for pull requests and make sure that no other message will influence contributors.

Reply:
Yes, there's an option. I'll try it locally. If it works, then we'll and good 👍. Else, need to find some other workaround.

Reply:
@AlbumenJ I'll let you know 🙂.